lib/pasarguard-restore.sh

```
@@ -0,0 +1,887 @@
+﻿#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] This file has a UTF-8 BOM. Remove it with: LC_CTYPE=C sed '1s/^...//' < yourscript .

(SC1082)